bentoml/_internal/models/model.py

```diff
@@ -0,0 +1,257 @@
+import logging
+import os
+import shutil
+import typing as t
+from collections import UserDict
+from datetime import datetime, timezone
+
+import attr
+import fs
+import fs.errors
+import fs.mirror
+import yaml
+from fs.base import FS
+from simple_di import Provide, inject
+
+from ...exceptions import BentoMLException
+from ..configuration import BENTOML_VERSION
+from ..configuration.containers import BentoMLContainer
+from ..store import Store, StoreItem
+from ..types import PathType, Tag
+
+if t.TYPE_CHECKING:
+    from ..runner import Runner
+
+logger = logging.getLogger(__name__)
+
+MODEL_YAML_FILENAME = "model.yaml"
+
+PathRes = t.TypeVar("PathRes", bound=t.Union[t.Optional[str], str])
+
+
+@attr.define(repr=False)
+class Model(StoreItem):
+    _tag: Tag
+    _fs: FS
+
+    _info: t.Optional["ModelInfo"] = None
+
+    @property
+    def tag(self) -> Tag:
+        return self._tag
+
+    def __eq__(self, other: "Model") -> bool:
+        return self._tag == other._tag
+
+    def __hash__(self) -> int:
+        return hash(self._tag)
+
+    @staticmethod
+    def create(
+        name: str,
+        *,
+        module: str = "",
+        labels: t.Optional[t.Dict[str, t.Any]] = None,
+        options: t.Optional[t.Dict[str, t.Any]] = None,
+        metadata: t.Optional[t.Dict[str, t.Any]] = None,
+        framework_context: t.Optional[t.Dict[str, t.Any]] = None,
+    ) -> "SysPathModel":
+        tag = Tag(name).make_new_version()
+        labels = {} if labels is None else labels
+        options = {} if options is None else options
+        metadata = {} if metadata is None else metadata
+        framework_context = {} if framework_context is None else framework_context
+
+        model_fs = fs.open_fs("temp://bentoml_model_{name}")
```

Comment:
This lines broke a lot of tests, seems that f string is wrongly formatted
```suggestion
        model_fs = fs.open_fs(f"temp://bentoml_model_{name}")
```
also the path tests look like this
```bash
/tmp/pytest-of-aarnphm/pytest-25/bentoml0/test_mlflow_save_load/nn5yl7cnffy6zkeafb746k2pu/tmp/tmpznwfm0fgbentoml_model_test_mlflow_save_load/sklearn_clf
```